lib/index.d.ts

```diff
@@ -1928,6 +1928,12 @@ declare namespace Joi {
         /**
          * Adds an alternative schema type for attempting to match against the validated value.
          */
+        try<A>(a: SchemaLike<A>): AlternativesSchema<A>;
+        try<A, B>(a: SchemaLike<A>, b: SchemaLike<B>): AlternativesSchema<A | B>;
+        try<A, B, C>(a: SchemaLike<A>, b: SchemaLike<B>, c: SchemaLike<C>): AlternativesSchema<A | B | C>;
+        try<A, B, C, D>(a: SchemaLike<A>, b: SchemaLike<B>, c: SchemaLike<C>, d: SchemaLike<D>): AlternativesSchema<A | B | C| D>;
+        try<A, B, C, D, E>(a: SchemaLike<A>, b: SchemaLike<B>, c: SchemaLike<C>, d: SchemaLike<D>, e: SchemaLike<E>): AlternativesSchema<A | B | C| D | E>;
+        try<A, B, C, D, E , F>(a: SchemaLike<A>, b: SchemaLike<B>, c: SchemaLike<C>, d: SchemaLike<D>, e: SchemaLike<E>, f: SchemaLike<F>): AlternativesSchema<A | B | C| D | E |F>;
```

Comment:
Looks like those triggered test errors, maybe what you want to use here is `SchemaLikeWithoutArray` like the original one?

Reply:
I don't see the failing tests, why do you think it is related to this type?

Reply:
You can see the failures in the CI runs, you should also get them locally. The errors are when passing arrays as argument, so it seems logical.

Reply:
Ok, I'll review them